at-quannguyen/W4/ex3.rb

```diff
@@ -0,0 +1,32 @@
+class ArrayClone
+  attr_accessor :array, :length
+  def initialize
+    self.array = []
+    self.length = 0
+  end
+
+  def map(&block)
+    result = []
+    @array.each { |e| result << yield(e) }
+    result
+  end
+
+  def select(&block)
+    result = []
+    @array.each { |e| result << i if yield(e) }
+    result
+  end
+
+  def length
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/6d0572eafbcb81a022ab0116c2936f3f51409ac2/at-quannguyen/W4/ex3.rb#L20"></span>Lint/DuplicateMethods: Method <code>ArrayClone#length</code> is defined at both at-quannguyen/W4/ex3.rb:2 and at-quannguyen/W4/ex3.rb:20.</td>
    </tr>
  </tbody>
</table>